docs/lib/graphqlapi/fragments/android/getting-started/20_installLib.md

```diff
@@ -0,0 +1,20 @@
+<!--TODO Update AWSmobile Client -> Auth -->
+Add the following dependencies to your **app** build.gradle file and click "Sync Now" when asked:
+
+```groovy
+dependencies {
+    implementation 'com.amplifyframework:core:0.10.0'
+    implementation 'com.amplifyframework:aws-api:0.10.0'
+}
+```
+
+Also at the top of the same file, add this piece of code to support the Java 8 features Amplify uses:
```

Comment:
I'd rather not duplicate this. Let's just point them to the project setup guide.

Reply:
this section has been revamped to latest version of Storage